doc/UserGuide.md

```diff
@@ -53,6 +53,10 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Listing all persons : `sort`
```

Comment:
Overall good job! Just one trivial point. Did you miss out on changing this line after you duplicated the code for 'list'? Your sort command doesn't return the list after sorting after all.

Reply:
Yeah I miss out on that haha. Thanks for pointing out